src/test/java/com/iota/iri/controllers/TransactionViewModelTest.java

```diff
@@ -396,47 +390,40 @@ public void testManyTXInDB() throws Exception {
     private Transaction getRandomTransaction(Random seed) {
         Transaction transaction = new Transaction();
 
-        int[] trits = Arrays.stream(new int[TransactionViewModel.SIGNATURE_MESSAGE_FRAGMENT_TRINARY_SIZE]).map(i -> seed.nextInt(3)-1).toArray();
+        byte[] trits = new byte[TransactionViewModel.SIGNATURE_MESSAGE_FRAGMENT_TRINARY_SIZE];
```

Comment:
`getRandomTransaction` impl. appears a few times, better to move to a Utils class.
but not part of this scope.

Reply:
👍